index.js

```diff
@@ -0,0 +1,147 @@
+const day = document.getElementById("js-day"),
+ date = document.getElementById("js-date"),
+ month = document.querySelector(".js-month");
+
+ today = new Date(),
+ MONTH = today.getMonth(),
+ YEAR = today.getFullYear(),
+ PAINTING = "painting";
```

Comment:
상수 처리를 하셨군요👍
단, 현재 변수 선언문이 없는데 선언문은 반드시 추가해주세요.
추후 변경되지 않는 값일 경우 `const`로 우선 선언 해주세요.

```suggestion
const today = new Date();
const MONTH = today.getMonth();
const YEAR = today.getFullYear();
const PAINTING = "painting";
```